test/LibraryManager.IntegrationTest/LibmanCompletionTests.cs

```diff
@@ -189,7 +194,8 @@ public void LibmanCompletion_VersionCompletionInDescendingOrder()
             Editor.Caret.MoveDown(1);
             Editor.KeyboardCommands.Type("{");
             Editor.KeyboardCommands.Enter();
-            Editor.KeyboardCommands.Type("\"provider\": \"unpkg\",");
+            Editor.KeyboardCommands.Type("\"provider\":");
+            Editor.KeyboardCommands.Type("unpkg\","); // auto-format will put the space & quote after the colon
```

Comment:
Nit: I'd maybe create a private TypeMember helper since we do it in so many places in this class

Reply:
I'm going to skip this for now.  Having too many different helpers for niche operations leads to a really confusing library (Apex is already bad enough with Editor.KeyboardCommands.Type vs Editor.Edit.InsertText vs Editor.Edit.InsertTextWithNewlines vs Editor.Edit.InsertTextInBuffer ...), and if we do add a helper it should be well designed to simplify things instead of adding just one more way to do things.  https://xkcd.com/927/